kubeflow/trainer/utils/utils.py

```
@@ -501,9 +502,7 @@ def get_args_in_dataset_preprocess_config(
     # Override the dataset source field if it is provided.
     if dataset_preprocess_config.source:
         if not isinstance(dataset_preprocess_config.source, types.DataFormat):
-            raise ValueError(
-                f"Invalid data format: {dataset_preprocess_config.source.value}."
-            )
+            raise ValueError(f"Invalid data format: {dataset_preprocess_config.source.value}.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)